examples/galaxy_tosca.yaml

```diff
@@ -55,6 +59,14 @@ topology_template:
       requirements:
         - lrms: local_lrms
 
+    galaxy_tools:
+      type: tosca.nodes.indigo.GalaxyShedTool
+      properties:
+        galaxy_flavor: { get_input: flavor }
+        admin_api_key: { get_input: admin_api_key }
```

Comment:
@mtangaro The nodetype tosca.nodes.indigo.GalaxyShedTool does not have this property you have to add it.